docs/language/ql-training/cpp/program-representation-cpp.rst

```diff
@@ -25,8 +21,8 @@ Agenda
 
 .. resume slides
 
-AST QL classes
-==============
+AST classes
```

Comment:
This is one file where I would retain the "QL" - otherwise new query authors can get confused with classes in the target language.

Reply:
`AST CodeQL classes` as they are specific to C++ analysis?